proj.spec

```diff
@@ -28,6 +28,7 @@ data with a wide range of selectable projection functions.
 %configure
 
 %build
+%{_builddir}/%{name}-%{version}/autogen.sh
```

Comment:
Okey, 
I will move the line just under the %prep tag, and try it out.

Thank you!